astropy/io/fits/column.py

```diff
@@ -832,8 +832,8 @@ def name(col, name):
 
         # This ensures that the new name can fit into a single FITS card
         # without any special extension like CONTINUE cards or the like.
-        if (not isinstance(name, str)
-                or len(str(Card('TTYPE', name))) != CARD_LENGTH):
```

Comment:
keep this one, the new guidelines put the binary operators to the beginning of the line rather than before a new line.

Reply:
Yes, but in io.fits the old guideline is used (this changed no so long ago), and I prefer to fix 14 lines for consistency (I will not fix manually the 208 lines to the "new" syntax):
```
❯ flake8 --select W503 astropy/io/fits/ | wc -l
14
❯ flake8 --select W504 astropy/io/fits/ | wc -l
208
```

Reply:
On the other hand autopep8 does it for you, so no need to do it manually.

Reply:
Also it's in pep8, so while having a strong preference might be only a few years old, it was always around as a recommendation

Reply:
_"For decades the recommended style was to break after binary operators"_
https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Also I never tried the automatic fixer in autopep8 for this, because it means breaking and  reformatting automatically some lines, maybe it can work.

Reply:
Ooof, I detest that new guideline, especially when it comes to logical operators in `if` statements.  It's much easier when reading, if the operator is at the end of the line, to process that as "ok, this predicate continues on the next line", otherwise it's easier to miss.